harness/determined/core/_log_shipper.py

```diff
@@ -89,7 +89,7 @@ class _ShutdownMessage:
 class _LogSender(threading.Thread):
     def __init__(self, session: api.Session, logs_metadata: Dict) -> None:
         self._queue = queue.Queue(maxsize=SHIP_QUEUE_MAX_SIZE)  # type: queue.Queue[_QueueElement]
-        self._logs = collections.deque()  # type: collections.deque[str]
+        self._logs = collections.deque()  # type: Deque[str]
```

Comment:
`typing.Deque` is technically "deprecated" https://docs.python.org/3/library/typing.html